nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-pubsub-processors/pom.xml

```diff
@@ -35,6 +35,10 @@
             <artifactId>nifi-utils</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-expression-language</artifactId>
```

Comment:
@ijokarumawak supporting EL for those properties makes sense. There are subject-less functions in EL as well as the ability to reference system, environment and custom variables set through the variable registry by using EL. 

I was just confused because since they already had references to EL functions, why did this need to get added as an explicit dependency?